packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.js

```diff
@@ -72,7 +72,7 @@ export default class QuanticBreadcrumbManager extends LightningElement {
 
   labels = {
     nMore,
-    clear,
+    clearFilter: clearFilter,
```

Comment:
it is the same name, so just  `clearFilter`

Reply:
oh yea, side-effect of using the refactor feature :D thanks!